tests/unit/test_charm.py

```diff
@@ -27,6 +28,9 @@ def setUp(self):
         self._postgresql_container = "postgresql"
         self._postgresql_service = "postgresql"
 
+        ops.testing.SIMULATE_CAN_CONNECT = True
```

Comment:
Should we commit globally? E.g:

```
tests/unit/__init__.py
-----
# Copyright 2022 Canonical Ltd.
# See LICENSE file for licensing details.

import ops.testing

ops.testing.SIMULATE_CAN_CONNECT = True
```

like https://github.com/canonical/mysql-k8s-operator/blob/main/tests/unit/__init__.py

Reply:
That's how charmcraft init should [generate](https://github.com/canonical/charmcraft/blob/main/charmcraft/templates/init-simple/tests/unit/test_charm.py.j2#L18) the tests (though I see now that I've forgot the cleanup). I'm fine with either way, but we should most probably adopt one as a convention.

Reply:
Sold. :-)